src/components/Checkbox/Checkbox.js

```diff
@@ -41,19 +85,21 @@ class Checkbox extends PureComponent<CheckboxProps> {
 
   render() {
     const { onChange, label, ...rest } = this.props;
+    const iconModifiers = { isShown: rest.checked || rest.indeterminate };
 
     return (
       <CheckboxWrapperTag { ...rest } tagName="label">
         <CheckboxSquareTag modifiers={ rest } tagName="div">
-          <CheckboxIconTag modifiers={ rest } checked={ rest.checked } tagName="div">
-            <Icon name="Check" size="xs" />
+          <CheckboxIconTag modifiers={ iconModifiers } tagName="div">
```

Comment:
@lesha1201 `rest` should be passed as `modifiers`

Reply:
@zouxuoz Can i do `const iconModifiers = { ...rest, isShown: rest.checked || rest.indeterminate };`?

Reply:
@lesha1201 yes, u can. We use `modifiers` to giving the ability to customize any component tag so we pass all rest props to this prop as a result user can customize any tag styles by passing custom theme to the provider.